subsys/bluetooth/controller/hci_driver.c

```diff
@@ -672,7 +672,6 @@ static int configure_memory_usage(void)
 	}
 #endif
 
-#if defined(CONFIG_BT_PERIPHERAL) || defined(CONFIG_BT_LL_SOFTDEVICE_MULTIROLE)
```

Comment:
I assume this will break central-only builds (the reason for merging this in the first place)

Reply:
Good point, I adjusted this now... Used the same logic we use to select the library to link agains: https://github.com/nrfconnect/sdk-nrfxlib/blob/d65788e1124b2babf1ce7aef85222eba82220b23/softdevice_controller/CMakeLists.txt#L24